### PR TITLE
added new ESH MQTT binding to distro

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
@davidgraeff FYI. I assume that is all that needs to be done to make it available in openHAB?

File: features/addons-esh/src/main/feature/feature.xml [hunk 1]
```diff
@@ -47,6 +47,11 @@
         <feature>esh-binding-lirc</feature>
     </feature>
 
+    <feature name="openhab-binding-mqtt" description="MQTT Binding" version="${project.version}">
```
Comment (cweitkamp) [inline features/addons-esh/src/main/feature/feature.xml]: Do we need dependency on `<feature>openhab-transport-mqtt</feature>`?

Comment (davidgraeff) [reply features/addons-esh/src/main/feature/feature.xml]: Definitely not. Esh mqtt transport is the new revamped one with asynchronous API and the openhab mqtt transport is the one from openHAB 1.x days.

**Comment by davidgraeff** [comment]
There are 2 mqtt PRs queuing up on the Esh side that will make the experience a lot smoother. But all in all yes I'm happy that we finally have this and I approve.

**Pull request merged**